library/Data/NBA/Stats.hs

```diff
@@ -158,6 +178,9 @@ instance Aeson.ToJSON Split where
         "headers" .= columns,
         "rowSet" .= rows]
 
+-- | A JSON response string
+type ResponseBody = LBS.ByteString
```

Comment:
I prefer `SplitData` or something similar to be consistent with the other aliases.